rgw/v2/tests/s3cmd/reusable.py

```diff
@@ -33,16 +33,17 @@ def create_bucket(bucket_name):
     assert expected_response in mb_response, error_message
 
 
-def upload_file(bucket_name):
+def upload_file(bucket_name, file_name=None):
     """
     Uploads file to the bucket
     Args:
         bucket_name(str): Name of the bucket
```

Comment:
As a new parameter is included, it is good to add docStrings for it.

Reply:
Done